css/bodystyle.css

```diff
@@ -0,0 +1,200 @@
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+}
+
+nav {
+  background-color: white;
+  height: 60px;
+  width: 100%;
+}
+
+img.logo {
+  width: 20px;
+  height: auto;
+  margin: 10px;
+  margin-left: 40px;
+}
+
+nav ul {
+  float: right;
+  margin: 0 20px;
+}
+
+nav ul li {
+  display: inline;
+  line-height: 60px;
+  margin: 0 5px;
+}
+
+nav ul li a {
+  color: rgb(68, 67, 67);
+  font-size: 20px;
+  padding: 5px 10px;
+}
+
+a:hover {
+  color: rgb(197, 58, 58);
+  transition: 0.5s;
+}
+
+.menubutton {
+  float: right;
+  color: rgb(68, 67, 67);
+  font-size: 20px;
+  line-height: 60px;
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+
+#check {
+  display: none;
+}
+
+main {
+  width: 100%;
+}
+/* HERO */
+#hero div {
+  min-height: 400px;
+  background: url("../img/first-background.jpg") no-repeat;
+  background-size: cover;
+  background-position: right;
+  color: whitesmoke;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: auto;
+  text-align: center;
+  padding: 20%;
+}
+
+#hero h2 {
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+}
+
+#hero p {
+  font-size: 15px;
+  margin: 10px;
+}
+
+#hero button {
+  margin-top: 20px;
+  width: 120px;
+  height: 40px;
+  font-size: 14px;
+  background-color: rgb(225, 106, 66);
+}
+
+/* Middle Title*/
+#middle {
+  margin-top: 40px;
+  padding: 10px;
+  align-items: center;
+}
+#middle p {
+  font-size: 20px;
+  text-align: center;
+  color: rgb(75, 74, 74);
+}
+
+/*Boxes*/
+#boxes div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: flex-start;
+}
+
+#boxes div .box {
+  width: 120px;
+  margin: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  width: 96%;
+  background-color: rgb(218, 211, 211);
+  height: 1px;
+  margin: 10px;
+}
+
+/*Footer*/
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 40px;
+  align-items: center;
+}
+
+footer .icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0.8;
+  width: 40px;
+  margin: 10px;
+  padding: 10px;
+  border-radius: 50%;
+  border: 1px solid rgb(209, 207, 207);
+}
+
+/*Hamburger Menu*/
+@media (max-width: 780px) {
+  .menubutton {
+    display: block;
+  }
+  ul {
+    position: fixed;
+    width: 50%;
+    height: auto;
+    background: whitesmoke;
+    top: 60px;
+    right: -100%;
+    text-align: center;
+    transition: all 0.5s;
+  }
```

Comment:
Love your mobile nav!
Perhaps give this `ul` a class to be a little more precise, and add your CSS to that instead of `ul` (a bit too generic).